### DISCUSSION TITLE
WebP support

### DISCUSSION THREAD

**Discussion by equinox2k** [SixLabors/ImageSharp#1953] [Ideas] [answered]
Hi,

Long time no speak glad to see Textures library is coming along whilst Ive been away...

Wanted to ask so I can plan a few work related things, WebP I assume is not in a main release yet, but is it expected to be in the V2 release I see mentioned for maybe Feb if all goes well?

**Comment by brianpopow** [comment]
> Wanted to ask so I can plan a few work related things, WebP I assume is not in a main release yet, but is it expected to be in the V2 release I see mentioned for maybe Feb if all goes well?

Yes that is still the plan. I am optimistic that we can make it, but I cannot guarantee it. There are still some issues open for V2.
 
> Long time no speak glad to see Textures library is coming along whilst Ive been away...

I did my best to continue the project. We have released a first alpha version to our MyGet feed! Still alot of work needs to be done there. While decoding of textures works well, encoding is still entirely missing. I was hoping to get some help from the community there.

**Reply by a2gil** [reply]
> I did my best to continue the project. We have released a first alpha version to our MyGet feed! Still alot of work needs to be done there. While decoding of textures works well, encoding is still entirely missing. I was hoping to get some help from the community there.

Hi there! i was looking for this support too and would be glad to contribute as well. Is it possible to have a look at your alpha version first?

**Reply by brianpopow** [reply]
You are talking about the alpha version of the Textures project, right? You can find the alpha version of the Textures project in our [MyGet Feed](https://www.myget.org/feed/sixlabors/package/nuget/SixLabors.ImageSharp.Textures). Make sure to enable `include prerelease` when you search for the package. 

The project itself can be found here: https://github.com/SixLabors/ImageSharp.Textures

Any PR are very welcome.

**Comment by brianpopow** [comment, marked as answer]
With release of 2.0.0, webp is now supported. 

Some features still missing: #1802
